src/main/java/uk/ac/ebi/tsc/tesk/config/security/AuthorisationProperties.java

```diff
@@ -43,6 +43,8 @@
      */
     private String adminGroupFull;
 
+    private boolean listUserJobsOnly;
```

Comment:
It might help, if you think of the change as `ignoreGroupMembership` and not `listUserJobsOnly`.
By default `ignoreGroupMembership` should be false and the system should work the same it does now -> a normal user in order to create tasks and see their own tasks needs to belong to at least one group.
If `ignoreGroupMemberhip` is on, each authenticated user can create tasks and they should be able to see their own tasks.

In the current code, if I understood it correctly, group membership requirement is switched off, regardless of the new parameter. And this is not what we want.